Day-II-Calculator/calculator/src/components/ButtonComponents/NumberButton.js

```diff
@@ -1,2 +1,18 @@
 import React from 'react';
-import './Button.css';
+//import './Button.css';
+
+function NumberButton(props) {
+  let className = 'number';
+  if (props.large) {
+  	className += ' number-large';
+  }
+  return (
+    
+	<React.Fragment>
+    	<div className={className}>{props.number}</div>
+    </React.Fragment>
```

Comment:
Why use a fragment here? It's unnecessary since it doesn't do anything for styling and since your div doesn't need a placeholder/pseudo parent. You can simply do this:
```javascript
return <div {...yourProps and/or attributes}>{yourContent}</div>;
```